webapp/src/utils/jira_issue_metadata.test.tsx

```diff
@@ -590,4 +590,99 @@ describe('utils/jira_issue_metadata', () => {
             expect(actual).toEqual('Project = KT AND IssueType IN (Bug) AND Priority IS EMPTY');
         });
     });
+
+    describe('jiraIssueToReducer', () => {
+        it('should return the ticket details with all fields', () => {
+            const action: Action = {
+                data: {
```

Comment:
It's unfortunate we need this unit test, when we could just store the jira issue itself in redux instead. It's more future-proof that way, as we only need to edit where it's being used, and not edit the reducer.

Reply:
Also, we can use more realistic test data from existing test data files https://github.com/mattermost/mattermost-plugin-jira/tree/master/webapp/src/testdata

Reply:
the data we require is not present in the testData directory. Do you want us to create a new file?

Reply:
Yes that sounds good